dask_cugraph/tests/test_hibench_small.py

```diff
@@ -0,0 +1,54 @@
+import operator
+from dask.distributed import Client, wait, default_client, futures_of
+from dask_cuda import LocalCUDACluster
+import gc
+from itertools import product
+import time
+import numpy as np
+
+import pytest
+# Temporarily suppress warnings till networkX fixes deprecation warnings
+# (Using or importing the ABCs from 'collections' instead of from
+# 'collections.abc' is deprecated, and in 3.8 it will stop working) for
+# python 3.7.  Also, this import networkx needs to be relocated in the
+# third-party group once this gets fixed.
+import warnings
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    import networkx as nx
+
+def test_pagerank():
+    gc.collect()
+    input_data_path = r"datasets/hibench_small/1/part-00000.csv"
+
+    # Networkx Call
+    import pandas as pd
+    pd_df = pd.read_csv(input_data_path, delimiter='\t', names=['src', 'dst'])
+    print(pd_df)
+    import networkx as nx
+    G = nx.Graph()
+    for i in range(0,len(pd_df)):
+        G.add_edge(pd_df['src'][i],pd_df['dst'][i])
+    nx_pr = nx.pagerank(G, alpha=0.85)
+    nx_pr = sorted(nx_pr.items(), key=lambda x: x[0])
+
+    # Cugraph snmg pagerank Call
+    cluster = LocalCUDACluster(threads_per_worker=1)
+    client = Client(cluster)
+    import numpy as np
+    import cudf
+    import dask_cudf
+    import dask_cugraph.pagerank as dcg
+    input_df = cudf.DataFrame()
+    print("Read Input Data.")
+    ddf = dcg.read_csv(input_data_path, delimiter='\t', names=['src', 'dst'], dtype=['int32', 'int32'])
+    print("CALLING DASK MG PAGERANK")
+    pr = dcg.mg_pagerank(ddf)
+    res_df = pr.compute()
+    err = 0
+    tol = 1.0e-02
```

Comment:
I think decreasing `tol` to `1.0e-05` and adding a number of iteration of _at least_ `50` to  `dcg.mg_pagerank` would make the test stronger.